src/styles/blocks/_card.scss

```
@@ -0,0 +1,24 @@
+.card {
+  text-align: center;
+
+  &__photo {
+    display: block;
+
```

effectively this should fix the mobile horizontal scroll
```suggestion
  &__photo {
    display: block;
    width: 100%;

```